src/pages/fr/guides/configuring-astro.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Configurer Astro
+---
+
+Personnalisez le fonctionnement d'Astro en ajoutant un fichier `astro.config.mjs` dans votre projet. Il s'agit d'un fichier commun aux projets Astro, et tous les modèles et thèmes d'exemple officiels en contiennent un par défaut.
+
+📚 Consultez la [référence d'API d'Astro](/fr/reference/configuration-reference/) pour une vue d'ensemble des options supportées.
+
+## Le Fichier de Configuration d'Astro
+
+Un fichier de configuration Astro valide exporte sa configuration en utilisant l'exportation `default`, et en utilisant l'aide recommandée `defineConfig` :
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config'
+
+export default defineConfig({
+  // vos options de configuration ici...
+  // https://docs.astro.build/fr/reference/configuration-reference/
+})
+```
+
+Utiliser `defineConfig()` est recommandé pour automatiser les types dans votre éditeur de code, mais c'est aussi optionnel. Un fichier de configuration absolument minimal, valide, pourrait ressembler à ça :
```

Comment:
Does "...pour automatiser les types..." _truly_ mean the same as "to receive automatic type hints"? It's not how I would expect to translate the phrase, but maybe it's some French usage that I'm not familiar with. I'm expecting some more like "...pour avoir/recevoir indications automatiques des types....".